### ISSUE TITLE
Error in inference on other size except (640,640) 

### ISSUE THREAD

**Issue by Yasin40** [Nebula4869/YOLOv5-LibTorch#7]
Hi,
I want inference my ```.torchscript.pt``` which is trained by this command and switch:
```
python train.py --img 128 128 --rect ...
```
but, by change resize function in this repo to (128,128), i got error:
```
RuntimeError: The size of tensor a (32) must match the size of tensor b (80) at non-singleton dimension 3
```

but, i don't know where should i change to work for me.
please help me.

**Comment by Nebula4869** [comment]
Bro, your error is obviously because there is a problem with the number of categories, and it has nothing to do with the image size. I used COCO to train 80 categories. Did you train a 32-category? Please do not come to me if there are any minor problems in the code, can it be solved by yourself?

**Comment by Yasin40** [comment]
No, if this problem was relate to category, doesn't work by change to 640,640. Am i right?

**Comment by Nebula4869** [comment]
Yes u r right, i have nothing to say




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "notifications"<notifications@github.com&gt;; 
发送时间: 2020年12月15日(星期二) 下午5:39
收件人: "Nebula4869/YOLOv5-LibTorch"<YOLOv5-LibTorch@noreply.github.com&gt;; 
抄送: "Nebula"<nebula4869@foxmail.com&gt;; "Comment"<comment@noreply.github.com&gt;; 
主题: Re: [Nebula4869/YOLOv5-LibTorch] Error in inference on other size except (640,640)  (#7)





 
No, if this problem was relate to category, doesn't work by change to 640,640. Am i right?
